src/DGtal/geometry/doc/moduleMetrics.dox

```diff
@@ -216,27 +216,29 @@ As discussed in @ref moduleVolumetric, both VoronoiMap and PowerMap (and their a
 subclasses) are parametrized by a generic separable metric (model of CSeparableMetric or CPowerSeparableMetric). If @f$ C
 @f$ corresponds to the cost of the @a closest() or @a
 closestPower() methods for the given metric, and @f$ H @f$ the cost
-of the @a hiddenBy() or @a hiddenByWeigthed(), the computational costs
-of the separable metrics can be summarized as follows:
+of the @a hiddenBy() or @a hiddenByWeigthed() in dimension @f$ d @f$, the computational costs
+of the separable metrics can be summarized as follows (for a domamain of size @f$ N^d @f$:
 
 
 | Models of CSeparableMetric/CPowerSeparableMetric | C | H | Note |
 |--------------------------------------------------|---|---|------|
-| ExactPredicateLpSeparableMetric<p>               | @f$ O(log(p)) @f$ | @f$ O(log(p)log(n)) @f$ | Exact computations|
-| ExactPredicateLpSeparableMetric specialized for p=2 | @f$ O(1) @f$ | @f$ O(1) @f$ | Exact computations|
-| ExactPredicateLpSeparableMetric specialized for p=1 | @f$ O(1) @f$ | @f$ O(log(n)) @f$ | Exact computations|
-| InexactPredicateLpSeparableMetric with p at construction | @f$ O(1) @f$ | @f$ O(log(n)) @f$ | Inexact computations since std::pow on double is used (supposed to be @f$ O(1) @f$)|
-| ExactPredicateLpPowerSeparableMetric<p> | @f$ O(log(p)) @f$ | @f$ O(log(p)log(n)) @f$ | Exact computations|
-| ExactPredicateLpPowerSeparableMetric specialized for p=2 | @f$ O(1) @f$ | @f$ O(1) @f$ | Exact computations|
-| SeparableMetricAdapter of a metric aMetric  | @f$ O(m) @f$ | @f$ O(mlog(n)) @f$ | relies on aMetric(p,q) computations |
-| experimental::ChamferNorm2D | @f$ O(log(p)) @f$ | @f$ O(log^2(n)) @f$ | @cite dcoeurjo_metric14|
+| ExactPredicateLpSeparableMetric<p>               | @f$ O(log(p)) @f$ | @f$ O(log(p)log(N)) @f$ | Exact computations|
+| ExactPredicateLpSeparableMetric specialized for p=2 | @f$ O(d) @f$ | @f$ O(d) @f$ | Exact computations|
+| ExactPredicateLpSeparableMetric specialized for p=1 | @f$ O(d) @f$ | @f$ O(d\cdot log(N)) @f$ | Exact computations (H could be implemented in @f$ O(d) @f$)|
```

Comment:
Usually \cdot is a scalar product for vector. I would just write O(d \log(N) ). The \log will make the formula nicer.

Reply:
\cdot is also just product on scalars when you want to separate the terms. With the product of several logs, it helps the readability I think

Reply:
As you wish !